__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require('supertest');
+const app = require('../app');
+
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+
+const {articleData, commentData, topicData, userData } = require('../db/data/test-data/index')
+
+beforeEach(()=>{
+    return seed({topicData, userData, articleData, commentData })
+})
+afterAll(() => {db.end()});
+
+describe('app', ()=>{
+    describe('api', ()=>{
+        describe('GET /api/topics', ()=>{
+            test('statuscode 200', ()=>{
+                return request(app).get('/api/topics').expect(200);
+            })
+            test('should return an array of topic objects', ()=>{
+                return request(app).get('/api/topics')
+                .then(({body})=>{
+                    const {topics} = body;
+                    expect(Array.isArray(topics)).toBe(true);
+                    topics.forEach((topic)=>{
+                        expect(typeof topic.slug).toBe('string');
+                        expect(typeof topic.description).toBe('string');
+                    }) 
```

Comment:
Great check here 🚀 

One note:
Whenever we do a forEach, we need to also make sure the array actually contains the number of entries that we expected. If the array returned was unexpectedly empty, the forEach loop wouldn't run and the test would still pass (nothing would fail) so we have to add another assertion to cover our backs. 

What could this check look like?